core/tx_list.go

```diff
@@ -292,6 +288,17 @@ func (l *txList) FeeCurrencies() []common.Address {
 	return feeCurrencies
 }
 
+func toCURRENCY(celoAmount *big.Int, feeCurrency *common.Address, txCtx *txPoolContext) (*big.Int, error) {
```

Comment:
Any reason for the unusual allcaps of CURRENCY? Might be for symmetry with `toCELO`, but since we don't use allcaps for "currency" in other places, we also shouldn't do so here.

Reply:
yeah just symmetry, changed it to Currency.